tests/test_authorizable.py

```diff
@@ -0,0 +1,73 @@
+#! pytest
+
+import pytest
+import eth_tester.exceptions
+
+
+def test_add_authorized_address(authorizable_contract, accounts, web3):
+    """test if address can be added by owner"""
+    contract = authorizable_contract
+    account_to_authorize = accounts[1]
+
+    assert contract.functions.authorized(account_to_authorize).call() is False
+
+    tx_hash = contract.functions.addAuthorizedAddress(account_to_authorize).transact({"from": accounts[0]})
+    tx_receipt = web3.eth.getTransactionReceipt(tx_hash)
+
+    assert tx_receipt is not None
+    assert contract.functions.authorized(account_to_authorize).call() is True
+
+
+def test_add_authorized_address_not_owner(authorizable_contract, accounts, web3):
+    """test onlyOwner modifier for addAuthorizedAddress"""
+    contract = authorizable_contract
+    other_account = accounts[1]
+
+    with pytest.raises(eth_tester.exceptions.TransactionFailed):
+        contract.functions.addAuthorizedAddress(other_account).transact({"from": other_account})
+
+
+def test_add_already_authorized_address(authorizable_contract, accounts, web3):
```

Comment:
should use `authorizable_contract_with_addresses` instead of `authorizable_contract` and remove the adding of `accounts[1]` as authorized. To make it more clear and consistent with following test.